openshift/e2e-tests-openshift.sh

```diff
@@ -178,21 +178,23 @@ function run_e2e_tests(){
   # adding the basic workflow tests for now
   # TODO: Link the integration tests written in go here once the PR is merged upstream
 
-  ./kn service create hello --image $KN_DEFAULT_TEST_IMAGE -e TARGET=Knative || failed=1
-  sleep 20
-  ./kn service get || failed=1
-  ./kn service update hello --env TARGET=kn || failed=1
-  sleep 20
-  ./kn revision get || failed=1
-  ./kn service get || failed=1
```

Comment:
@rhuss : var `failed` was getting overridden with following commands' success thus, it didnt report the failure since a while (we renamed `get` to `list` a while ago).

Also we'll need to get the e2e tests in shape.